jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcWorkerJobRunningRepository.java

```diff
@@ -0,0 +1,92 @@
+package io.kestra.jdbc.repository;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.kestra.core.repositories.WorkerJobRunningInterface;
+import io.kestra.core.runners.WorkerJobRunning;
+import io.kestra.core.runners.WorkerTaskRunning;
+import io.kestra.core.runners.WorkerTriggerRunning;
+import io.kestra.core.serializers.JacksonMapper;
+import jakarta.inject.Singleton;
+import lombok.extern.slf4j.Slf4j;
+import org.jooq.DSLContext;
+import org.jooq.Record1;
+import org.jooq.SelectConditionStep;
+import org.jooq.impl.DSL;
+
+import java.util.List;
+import java.util.Optional;
+
+@Singleton
+@Slf4j
+public abstract class AbstractJdbcWorkerJobRunningRepository extends AbstractJdbcRepository implements WorkerJobRunningInterface {
+    protected io.kestra.jdbc.AbstractJdbcRepository<WorkerJobRunning> jdbcRepository;
+    private static final ObjectMapper MAPPER = JacksonMapper.ofJson();
```

Comment:
```suggestion
    private static final ObjectMapper MAPPER = JacksonMapper.ofJson();
    
    protected io.kestra.jdbc.AbstractJdbcRepository<WorkerJobRunning> jdbcRepository;
```

Nitpicking here but the declaration order is constant first.